tests/onnx/quantization/test_classification_models_graph.py

```diff
@@ -23,32 +25,37 @@
 from tests.onnx.weightless_model import load_model_topology_with_zeros_weights
 
 TORCHVISION_TEST_DATA = [
-    (ModelToTest("resnet18", [1, 3, 224, 224]), models.resnet18(pretrained=True)),
-    (ModelToTest("mobilenet_v2", [1, 3, 224, 224]), models.mobilenet_v2(pretrained=True)),
-    (ModelToTest("mobilenet_v3_small", [1, 3, 224, 224]), models.mobilenet_v3_small(pretrained=True)),
-    (ModelToTest("inception_v3", [1, 3, 224, 224]), models.inception_v3(pretrained=True)),
-    (ModelToTest("googlenet", [1, 3, 224, 224]), models.googlenet(pretrained=True)),
-    (ModelToTest("vgg16", [1, 3, 224, 224]), models.vgg16(pretrained=True)),
-    (ModelToTest("shufflenet_v2_x1_0", [1, 3, 224, 224]), models.shufflenet_v2_x1_0(pretrained=True)),
-    (ModelToTest("squeezenet1_0", [1, 3, 224, 224]), models.squeezenet1_0(pretrained=True)),
-    (ModelToTest("densenet121", [1, 3, 224, 224]), models.densenet121(pretrained=True)),
-    (ModelToTest("mnasnet0_5", [1, 3, 224, 224]), models.mnasnet0_5(pretrained=True)),
+    (ModelToTest("resnet18", [1, 3, 224, 224]), models.resnet18(pretrained=True), {}),
+    (
+        ModelToTest("resnet50", [1, 3, 224, 224]),
```

Comment:
You gave a name to the model resnet50, while using resnet18.
Moreover, the name does not reflect the quantization parameters used.

Reply:
Thx, fixed.